clusters/clusters_api.go

```diff
@@ -428,48 +428,48 @@ type ClusterList struct {
 	DriverInstancePoolID      string             `json:"driver_instance_pool_id,omitempty" tf:"computed"`
-	PolicyID                  string             `json:"policy_id,omitempty"`
-	SingleUserName            string             `json:"single_user_name,omitempty"`
-	ClusterSource             Availability       `json:"cluster_source,omitempty"`
-	DockerImage               *DockerImage       `json:"docker_image,omitempty"`
-	State                     ClusterState       `json:"state"`
-	StateMessage              string             `json:"state_message,omitempty"`
-	StartTime                 int64              `json:"start_time,omitempty"`
-	TerminateTime             int64              `json:"terminate_time,omitempty"`
-	LastStateLossTime         int64              `json:"last_state_loss_time,omitempty"`
-	LastActivityTime          int64              `json:"last_activity_time,omitempty"`
-	ClusterMemoryMb           int64              `json:"cluster_memory_mb,omitempty"`
-	ClusterCores              float64            `json:"cluster_cores,omitempty"`
-	DefaultTags               map[string]string  `json:"default_tags"`
-	ClusterLogStatus          *LogSyncStatus     `json:"cluster_log_status,omitempty"`
-	TerminationReason         *TerminationReason `json:"termination_reason,omitempty"`
-	DataSecurityMode          string             `json:"data_security_mode,omitempty"`
+	PolicyID                  string             `json:"policy_id,omitempty" tf:"computed"`
+	SingleUserName            string             `json:"single_user_name,omitempty" tf:"computed"`
+	ClusterSource             Availability       `json:"cluster_source,omitempty" tf:"computed"`
+	DockerImage               *DockerImage       `json:"docker_image,omitempty" tf:"computed"`
+	State                     ClusterState       `json:"state,omitempty" tf:"computed"`
+	StateMessage              string             `json:"state_message,omitempty" tf:"computed"`
+	StartTime                 int64              `json:"start_time,omitempty" tf:"computed"`
+	TerminateTime             int64              `json:"terminate_time,omitempty" tf:"computed"`
+	LastStateLossTime         int64              `json:"last_state_loss_time,omitempty" tf:"computed"`
+	LastActivityTime          int64              `json:"last_activity_time,omitempty" tf:"computed"`
+	ClusterMemoryMb           int64              `json:"cluster_memory_mb,omitempty" tf:"computed"`
+	ClusterCores              float64            `json:"cluster_cores,omitempty" tf:"computed"`
+	DefaultTags               map[string]string  `json:"default_tags,omitempty" tf:"computed"`
+	ClusterLogStatus          *LogSyncStatus     `json:"cluster_log_status,omitempty" tf:"computed"`
+	TerminationReason         *TerminationReason `json:"termination_reason,omitempty" tf:"computed"`
+	DataSecurityMode          string             `json:"data_security_mode,omitempty" tf:"computed"`
```

Comment:
create new type for cluster data resource and probably just expose cluster info as a field. this is the least intrusive change.